packages/trader/src/AppV2/Components/ContractDetailsFooter/contract-details-footer.tsx

```diff
@@ -82,7 +82,9 @@ const ContractDetailsFooter = observer(({ contract_info }: ContractInfoProps) =>
                 <Button
                     label={
                         is_valid_to_sell
-                            ? `${cardLabels.CLOSE} @ ${FormatUtils.formatMoney(bid_price || 0)} ${currency}`
+                            ? `${cardLabels.CLOSE} @ ${FormatUtils.formatMoney(bid_price || 0, {
+                                  currency: currency as any, // currency types mismatched between utils and shared
```

Comment:
Maybe other type assertion will work? Like `currency as 'USD'` or any other from list below
<img width="958" alt="Screenshot 2024-07-23 at 11 39 44 AM" src="https://github.com/user-attachments/assets/a38c089d-875c-47aa-bf27-45969c7c02b3">

Reply:
Makes zero sense to do it, I think just having `any` is better option

Reply:
Because, it doesn't make sense to update this type every time there's any currency change to me. I'd use it if it was something less strange.